src/React.re

```diff
@@ -390,3 +390,10 @@ external useImperativeHandle7:
 
 [@bs.set]
 external setDisplayName: (component('props), string) => unit = "displayName";
+
+[@bs.deriving abstract]
```

Comment:
Note here for me mostly - drop this if we end up bumping BS min version to record as obj support.